packages/zapp/console/src/components/Executions/Tables/nodeExecutionColumns.tsx

```diff
@@ -139,16 +139,13 @@ export function generateColumns(
       label: 'type',
     },
     {
-      cellRenderer: ({
-        execution: {
-          closure: { phase = NodeExecutionPhase.UNDEFINED, taskNodeMetadata },
-        },
-      }) => (
+      cellRenderer: ({ execution }) => (
         <>
-          <ExecutionStatusBadge phase={phase} type="node" />
-          {hasCacheStatus(taskNodeMetadata) ? (
-            <NodeExecutionCacheStatus taskNodeMetadata={taskNodeMetadata} variant="iconOnly" />
-          ) : null}
+          <ExecutionStatusBadge
+            phase={execution.closure?.phase || NodeExecutionPhase.UNDEFINED}
```

Comment:
replace || with ?? . It's a good practice to use ?? instead of ||, as || could lead to nonobvious errors which are harder to catch.
`execution.closure?.phase ?? NodeExecutionPhase.UNDEFINED`